config/pipelines/pipeline.conf

```diff
@@ -123,6 +123,27 @@ sources=[
     "source": {
       type="com.arpnetworking.metrics.common.sources.CollectdHttpSourceV1"
       name="collectd_http_source"
+    },
+    {
+      type="com.arpnetworking.metrics.common.sources.KafkaSource"
+      name="kafka_source"
+      consumer={
+        type="org.apache.kafka.clients.consumer.Consumer"
+        topics=["topic"]
+        configs={
+          # Set any properties defined at: https://kafka.apache.org/documentation/#consumerconfigs
+          bootstrap.servers="localhost:9092"
+          group.id="group0"
+          client.id="consumer0"
+          key.deserializer="org.apache.kafka.common.serialization.StringDeserializer"
```

Comment:
As I understand it, this configures how Kafka reads data out of the message. This doesn't seem appropriate if we're dealing with Protobufs. Wouldn't something that reads into a byte array be safer to avoid mangling due to character set interpretation?

https://kafka.apache.org/10/javadoc/org/apache/kafka/common/serialization/ByteArrayDeserializer.html